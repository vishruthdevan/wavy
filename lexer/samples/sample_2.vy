audioFiles = ["input1.wav", "input2.wav", "input3.wav"]
audioClips = []

foreach file in audioFiles {
    clip = load(file)
    fadeIn(clip, 3.0)
    append(audioClips, clip)
}

total_duration = getDuration(audioClips[0]) + getDuration(audioClips[1]) + getDuration(audioClips[2])