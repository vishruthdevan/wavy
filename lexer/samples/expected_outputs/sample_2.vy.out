<IDENTIFIER, "audioFiles">
<ASSIGN, "=">
<LBRACKET, "[">
<STRING, "input1.wav">
<COMMA, ",">
<STRING, "input2.wav">
<COMMA, ",">
<STRING, "input3.wav">
<RBRACKET, "]">
<IDENTIFIER, "audioClips">
<ASSIGN, "=">
<LBRACKET, "[">
<RBRACKET, "]">
<FOREACH, "foreach">
<IDENTIFIER, "file">
<IN, "in">
<IDENTIFIER, "audioFiles">
<LBRACE, "{">
<IDENTIFIER, "clip">
<ASSIGN, "=">
<IDENTIFIER, "load">
<LPR, "(">
<IDENTIFIER, "file">
<RPR, ")">
<IDENTIFIER, "fadeIn">
<LPR, "(">
<IDENTIFIER, "clip">
<COMMA, ",">
<FLOAT, "3.0">
<RPR, ")">
<IDENTIFIER, "append">
<LPR, "(">
<IDENTIFIER, "audioClips">
<COMMA, ",">
<IDENTIFIER, "clip">
<RPR, ")">
<RBRACE, "}">
<IDENTIFIER, "total_duration">
<ASSIGN, "=">
<IDENTIFIER, "getDuration">
<LPR, "(">
<IDENTIFIER, "audioClips">
<LBRACKET, "[">
<INTEGER, "0">
<RBRACKET, "]">
<RPR, ")">
<PLUS, "+">
<IDENTIFIER, "getDuration">
<LPR, "(">
<IDENTIFIER, "audioClips">
<LBRACKET, "[">
<INTEGER, "1">
<RBRACKET, "]">
<RPR, ")">
<PLUS, "+">
<IDENTIFIER, "getDuration">
<LPR, "(">
<IDENTIFIER, "audioClips">
<LBRACKET, "[">
<INTEGER, "2">
<RBRACKET, "]">
<RPR, ")">
<EOF, "">
